h/db.py

```diff
@@ -62,6 +63,8 @@ def bind_engine(engine,
     session.configure(bind=engine)
     base.metadata.bind = engine
     if should_drop:
+        transaction.commit()
```

Comment:
That's there to close any db connections, without it our model tests just hang forever